### DISCUSSION TITLE
Changing a Query (MySQL) doesnt update Data Providers bound to it

### DISCUSSION THREAD

**Discussion by poirazis** [Budibase/budibase#7682] [General]
Reproduce : 

Create a MySQL connection.

(assume the below tables / fields are defined)

Create a Query with a binding -> "SELECT * FROM members WHERE dept_id = {{ dept_id }} "
Create a Screen
Insert Data Provider
Set Data Provider Source to above query

Go back to "Data"
Change the Query, example "SELECT name, surname FROM members WHERE dept_id = {{ dept_id }} "
Go back to the Screen
Check Data Provider Source 
In the Panel that will open below, you will see the previous definition of the query.
In order to update, you need to select the query again in the Data Providers source drop down.

Is this expected behavior or a bug ?

**Comment by melohagan** [comment]
Hey @poirazis 

I was able to reproduce this bug, thanks for raising! 

https://github.com/Budibase/budibase/issues/7754
